lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -615,17 +615,17 @@ namespace Js
                         recycler->ReportExternalMemoryFailure(length);
                     }
                 }
-                else
-                {
-                    JavascriptError::ThrowOutOfMemoryError(GetScriptContext());
-                }
             }
 
             if (buffer != nullptr)
             {
                 bufferLength = length;
                 ZeroMemory(buffer, bufferLength);
             }
+            else
```

Comment:
This was probably an error made in https://github.com/Microsoft/ChakraCore/commit/c04bab38bf0d2f09c7e2cc98b200527418321ade#diff-699ac297c4c6339e45f156e1316de619